mscore/pa.cpp

```diff
@@ -73,6 +73,9 @@ Portaudio::Portaudio(Seq* s)
 Portaudio::~Portaudio()
       {
       if (initialized) {
+            if (midiDriver)
+                  delete midiDriver;
```

Comment:
Does it fix mem leak?

Reply:
Yes.  The driver wasn't being deleted previously with left the audio channel open and prevented a new one being created.